CRM/Core/DAO/Domain.php

```diff
@@ -173,15 +173,11 @@ static function &fields() {
           'type' => CRM_Utils_Type::T_TEXT,
           'title' => ts('Domain Configuration') ,
           'description' => 'Backend configuration.',
-          'rows' => 20,
```

Comment:
How come we lost html definition off these fields?

Reply:
Cause it's not an html field in any sense that I can tell. The html tags were originally added by someone with not much familiarity with the db and they're still not used for much of anything so I think it's safe to remove this.

Reply:
ok